### PR TITLE
[VM] SyndieDrobe

### PR DESCRIPTION
**Pull request by PuroSlavKing**
image

**Animation:**

https://github.com/space-wizards/space-station-14/assets/103608145/a480dac4-c551-40ca-8dfa-013c44ede352

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl: PuroSlavKing
- add: Added SyndieDrobe!

### CODE REVIEW

File: Resources/Locale/en-US/advertisements/vending/syndiedrobe.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+advertisement-syndiedrobe-1 = Brand new outfits!
+advertisement-syndiedrobe-2 = Roof-top outfits for any occasion!
+advertisement-syndiedrobe-3 = Being a scoundrel can be stylish.
+advertisement-syndiedrobe-4 = According to the analysis: by dressing more stylishly, the chance of success of your actions increases by 0.0098%!
+advertisement-syndiedrobe-5 = Hey, it seems to me you haven't looked at my assortment for a long time!
+advertisement-syndiedrobe-6 = Death of NT!
```
Comment (deltanedas) [inline Resources/Locale/en-US/advertisements/vending/syndiedrobe.yml:6]: of -> to

**Review by deltanedas — CHANGES_REQUESTED** [review]

File: Resources/Locale/en-US/advertisements/vending/syndiedrobe.yml (hunk 1 above)
Comment (PuroSlavKing) [reply Resources/Locale/en-US/advertisements/vending/syndiedrobe.yml]: > of -> to

done

File: Resources/Prototypes/Catalog/VendingMachines/Inventories/syndiedrobe.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    ClothingUniformJumpsuitSyndieFormalDress: 5
+    ClothingUniformJumpskirtSyndieFormalDress: 5
+    ClothingUniformJumpsuitOperative: 5
+    ClothingUniformJumpskirtOperative: 5
+    ClothingNeckScarfStripedSyndieGreen: 2
+    ClothingNeckScarfStripedSyndieRed: 2
+  emaggedInventory:
+    ClothingOuterCoatSyndieCapArmored: 1
+    ClothingOuterWinterSyndieCapArmored: 1
+    ClothingHeadHatTacticalMaidHeadband: 5
```
Comment (CaptainSqrBeard) [inline Resources/Prototypes/Catalog/VendingMachines/Inventories/syndiedrobe.yml:20]: please don't

Comment (PuroSlavKing) [reply Resources/Prototypes/Catalog/VendingMachines/Inventories/syndiedrobe.yml:20]: 😈

**Pull request merged**